Sources/URLSessionDecodable/Parameters.swift

```diff
@@ -45,17 +45,16 @@ public struct JSONParametersEncoder: ParametersEncoding {
 // MARK: - URL Encoding
 
 /// Encodes parameters adding them to the URL.
-public struct URLParametersEncoder<Value: CustomStringConvertible>: ParametersEncoding {
+public struct URLParametersEncoder<Parameters>: ParametersEncoding where Parameters: Collection<(key: String, value: CustomStringConvertible)> {
```

Comment:
nit: it wouldn't hurt to make generic value too: `Value: CustomStringConvertible`

Reply:
hmm, if we do that, then we won't be able to have different types in one collection, like that.
<img width="526" alt="image" src="https://github.com/ViacomInc/URLSessionDecodable/assets/138483245/b45dcb69-fb01-4a1d-be53-44c3d7ebe999">

Reply:
good point, let's leave it the way it is then